UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -113,12 +113,6 @@ public void SetUpSpawnablePrefabsForEverID()
 	void IInitialise.Initialise()
 	{
 		ApplyConfig();
-
-		var prevEditorScene = SubSceneManager.GetEditorPrevScene();
-		if (prevEditorScene != string.Empty && prevEditorScene != "StartUp" && prevEditorScene != "Lobby")
-		{
-			StartHost();
```

Comment:
where are you starting host now?

Reply:
Wdym? What does this piece of code do in the first place?

Reply:
I have no idea but whatever it did, it isn't doing it anymore. That's scary.

Reply:
I'll go back a commit or two and see what was the point of it later, for now I want to focus on the bundle download issue and re-locate where our executables read and store bundles.

Reply:
honestly, don't bother about remote distribution just yet. Let's use local bundles and move then to streamming assets during build. The zip file players download will be the same size, but we can figure out the best way to do remote distribution later.